main.tex

```diff
@@ -631,34 +635,33 @@ \section{Discussion}
         (Theorem~\ref{memone_group_best_response}).
 \end{itemize}
 
-Note that Theorem~\ref{memone_group_best_response} which does not only have game theoretic
-novelty, but also the mathematical novelty of solving quadratic ratio optimisation
-problems where the quadratics are non concave.
+Theorem~\ref{theorem_one} has to a further theoretical result of identifying
```

Comment:
I don't know what you are trying to say here.

Reply:
There is one further theoretical result that can be obtained from Theorem~\ref{theorem_one}.